ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -53,6 +53,8 @@ ${NOTIFICATION_DRAWER_CLOSED}=  //div[@class="pf-v5-c-drawer__panel" and @hidden
 ${GROUPS_CONFIG_CM}=    groups-config
 ${RHODS_GROUPS_CONFIG_CM}=    rhods-groups-config
 ${RHODS_LOGO_XPATH}=    //img[@alt="${ODH_DASHBOARD_PROJECT_NAME} Logo"]
+${USER_MENU_TOGGLE}=    //button[@id="user-menu-toggle"]
+${LOGOUT_BTN}=    //button[.="Log out"]
```

Comment:
does this depend on the version of OpenShift in use?
![image](https://github.com/user-attachments/assets/b5a490b1-1497-4d8b-8a60-b49262a30d13)

Reply:
never mind, this is a different log out